vmc/utils_test.go

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2022 VMware, Inc.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package vmc
+
+import (
+	"github.com/vmware/vsphere-automation-sdk-go/services/vmc/model"
+	"testing"
+)
+
+func TestToHostInstanceType(t *testing.T) {
```

Comment:
Use a table driven test instead. You can also run in parallel: see https://dave.cheney.net/2019/05/07/prefer-table-driven-tests and https://www.gopherguides.com/articles/table-driven-testing-in-parallel